src/settings/components/OneFieldSetting.svelte

```diff
@@ -15,5 +15,11 @@
 </script>
 
 <BaseSetting {title} {description}>
-  <vscode-text-field {value} on:change={onChange} />
+  <vscode-text-field class="vscode-text-field" {value} on:change={onChange} />
 </BaseSetting>
+
+<style>
+  .vscode-text-field {
```

Comment:
Could you please explain what you use this for?

Reply:
yes. I thought the field box was a too small because it should get a full path to the solc executable and it seems weird to have a long text input inside such small box.